continuousflex/protocols/protocol_mdspace.py

```diff
@@ -402,7 +382,7 @@ def _validate(self):
         return errors
 
     def _citations(self):
-        return ['harastani2022continuousflex','vuillemot2022NMMD']
+        return ['harastani2022continuousflex','vuillemot2022NMMD','vuillemot2023mdspace']
```

Comment:
You may actually want to revert the order of these citations.

Reply:
Done